specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json

```diff
@@ -1666,11 +1705,11 @@
         "modelSettings": {
           "description": "Model settings including model name, Time Series ID properties and default type ID.",
           "readOnly": true,
-          "$ref": "#/definitions/ModelSettings"
+          "$ref": "#/definitions/TimeSeriesModelSettings"
```

Comment:
I would keep original. We don't add TimeSeries to other data structures... or does it collide during code gen? Then I think the change is right.

Reply:
Yes, It collides with the ModelSettings_ operation group.